lib/foreman/logging.rb

```diff
@@ -86,7 +86,7 @@ def exception(context_message, exception, options = {})
       options.assert_valid_keys :level, :logger
       logger_name = options[:logger] || 'app'
       level = options[:level] || :warn
-      backtrace_level = options[:backtrace_level] || :debug
+      backtrace_level = options[:backtrace_level] || :warn
```

Comment:
I think info level is probably ok, in case someone has something that is flooding their logs with backtraces, but no strong preference.

Reply:
Why are we doing this? Please keep debug, stacktraces should not be shown at info or higher levels they are very noisy.

Reply:
The problem is that I would like to save a stacktrace to logfile because after I can search it. Do you have better idea how to  achieve this?

Reply:
This should be `:info` imho so users who really don't care can set it to `:warn`. We are doing this because otherwise all errors are swallowed in the logs and become impossible to debug, now that we are removing the stacktrace from the error page.
If there is some place that causes the log to flood with errors we should fix that, not hide the errors.

Reply:
I don't agree with `info` because backtraces can be lengthy and when something bad happens log files can easily grow by order of magnitude just because there's much more info logged. The message is logged at info level already and that should be usually enough, we need the backtrace in 1 of 10 cases.

Reply:
I strongly disagree. The default logging level is info, meaning no backtraces are logged. This is fine when we display the stacktrace to the user, but if we don't display them we have to log them at info. In my experience most bug reports are almost useless without a backtrace.

Reply:
Whatever, it makes more sense to show them as we advice for searching them.